h5py/_hl/files.py

```diff
@@ -449,6 +452,19 @@ def __init__(self, name, mode='r', driver=None, libver=None, userblock_size=None
             Warning: The HDF5_USE_FILE_LOCKING environment variable can override
             this parameter.
             Only available with HDF5 >= 1.12.1 or 1.10.x >= 1.10.7.
+
+        alignment_threshold
+            Together with ``alignment_interval``, this property ensures that
+            any file object greater than or equal in size to the alignement
+            threshold (in bytes) will be aligned on an address which is a
+            multiple of alignment interval.
+
+        alignment_interval
+            This property should be used in conjunction with
+            ``alignment_threshold``. See the description above. For more
+            details, see
+            https://support.hdfgroup.org/HDF5/doc/RM/RM_H5P.html#Property-SetAlignment
```

Comment:
```suggestion
            https://portal.hdfgroup.org/display/HDF5/H5P_SET_ALIGNMENT
```

The [support.hdfgroup.org](https://support.hdfgroup.org/) pages are no longer maintained.